tests/dryrun_test.go

```diff
@@ -149,34 +148,29 @@ var _ = Describe("[sig-compute]Dry-Run requests", decorators.SigCompute, func()
 	})
 
 	Context("VirtualMachines", func() {
-		var vm *v1.VirtualMachine
-		resource := "virtualmachines"
-
-		newVM := func() *v1.VirtualMachine {
-			vmiImage := cd.ContainerDiskFor(cd.ContainerDiskCirros)
-			vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(vmiImage, "echo Hi\n")
-			vm := libvmi.NewVirtualMachine(vmi)
-			return vm
-		}
+		var (
+			vm        *v1.VirtualMachine
+			namespace string
```

Comment:
Who is placing a value in this variable? I somehow cannot see it.

Reply:
removed by mistake